sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -875,15 +875,18 @@ trait CheckAnalysis extends LookupCatalog with QueryErrorsBase with PlanToString
             !o.isInstanceOf[CollectMetrics] &&
             !o.isInstanceOf[Filter] &&
             !o.isInstanceOf[Aggregate] &&
+            // The rule above is used to check Aggregate operator.
```

Comment:
This comment previously added in https://github.com/apache/spark/commit/e0701c75601c43f69ed27fc7c252321703db51f2 but it seemed to refer to Aggregate operator, and got out of sync with the call, so putting it back (although tbh its not a super helpful comment)